### PR TITLE
[14.0][MIG] purchase_reception_notify: Migration to 14.0

### PR DESCRIPTION
**Pull request by daylinglez**
@AaronHForgeFlow @JordiBForgeFlow could you please review?

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional on runbot :+1:

**Comment by daylinglez** [comment]
@HviorForgeFlow could you review? :wink:

File: purchase_reception_notify/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            message += _("<li><b>%s</b>: Received quantity %s %s</li>") % (
+                purchase_line_id["purchase_line"].product_id.display_name,
+                purchase_line_id["stock_move"].product_qty,
+                purchase_line_id["stock_move"].product_uom.name,
+            )
+        message += "</ul>"
+        return message
+
+    def _action_done(self):
+        super(StockPicking,self)._action_done()
```
Comment (HviorForgeFlow) [inline purchase_reception_notify/models/stock_picking.py]: Precommit failing due to a missing space, you could add the space or do this:
```suggestion
        super()._action_done()
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_reception_notify/models/stock_picking.py (hunk 1 above)
Comment (HviorForgeFlow) [reply purchase_reception_notify/models/stock_picking.py]: Run pre-commit again! there is another issue here: https://github.com/OCA/purchase-workflow/pull/1174/checks?check_run_id=2607277868#step:4:105

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**